workers/tasks/import/bulk/directives/pub.ts

```diff
@@ -154,7 +154,26 @@ const resolveDirectiveValues = async (directive, sourceFiles, rawMetadata, doc):
 	await Promise.all(
 		Object.entries(directive).map(async ([key, value]) => {
 			const resolvedValue = await resolveDirectiveValue(value, context);
-			resolvedDirective[key] = resolvedValue;
+			if (key === 'downloads') {
+				resolvedDirective[key] = [{ url: resolvedValue, type: 'formatted' }];
+			} else if (key === 'inlineFile') {
+				const { $sourceFile } = value as any;
+				const resolvedFileName =
+					`${rawMetadata.title}${resolvedValue
+						.split('/')
+						[resolvedValue.split('/').length - 1].substring(
+							resolvedValue.indexOf('.'),
+						)}` ||
+					$sourceFile ||
+					resolvedValue.split('/')[resolvedValue.split('/').length - 1];
```

Comment:
Could you pull `resolvedValue.split('/').length - 1` into a const above? Would make this a lot more readable I think